java-squid/src/main/java/org/sonar/java/se/ProgramState.java

```diff
@@ -40,11 +40,23 @@
 
 public class ProgramState {
 
+  public static class ConstrainedValue {
+
+    public final SymbolicValue value;
+    public final ObjectConstraint constraint;
+
+    ConstrainedValue(SymbolicValue value, ObjectConstraint constraint) {
+      super();
```

Comment:
unecessary call to super but IMO this class should be removed see comment https://github.com/SonarSource/sonar-java/pull/538/files#r47924681